shared/common-adapters/usernames/index.js

```diff
@@ -1,80 +1,84 @@
-      ...(colorFollowing && !u.you ? {color: u.following ? globalColors.green2 : globalColors.blue} : null),
-      ...(colorBroken && u.broken && !u.you ? {color: redColor || globalColors.red} : null),
-      ...(inline && !isMobile ? {display: 'inline'} : null),
-      ...(u.you ? globalStyles.italic : null),
-      ...(colorYou && u.you
-        ? {color: typeof colorYou === 'string' ? colorYou : globalColors.black_75}
+      ...(!Styles.isMobile ? {textDecoration: 'inherit'} : null),
+      ...(props.colorFollowing && !u.you
+        ? {color: u.following ? Styles.globalColors.green2 : Styles.globalColors.blue}
+        : null),
+      ...(props.colorBroken && u.broken && !u.you
+        ? {color: props.redColor || Styles.globalColors.red}
+        : null),
+      ...(props.inline && !Styles.isMobile ? {display: 'inline'} : null),
+      ...(u.you ? Styles.globalStyles.italic : null),
+      ...(props.colorYou && u.you
+        ? {color: typeof props.colorYou === 'string' ? props.colorYou : Styles.globalColors.black_75}
         : null),
     }
-    userStyle = collapseStyles([style, userStyle])
+    userStyle = Styles.collapseStyles([props.style, userStyle])
 
     // Make sure onClick is undefined when _onUsernameClicked is, so
     // as to not override any existing onClick handler from containers
     // on native. (See DESKTOP-3963.)
-    const _onUsernameClicked = onUsernameClicked
+    const _onUsernameClicked = props.onUsernameClicked
     return (
-      <Text type={type} key={u.username}>
+      <Text type={props.type} key={u.username}>
         {i !== 0 &&
-          i === users.length - 1 &&
-          showAnd && (
-            <Text type={type} backgroundMode={backgroundMode} style={derivedJoinerStyle}>
+          i === props.users.length - 1 &&
+          props.showAnd && (
+            <Text type={props.type} backgroundMode={props.backgroundMode} style={derivedJoinerStyle}>
               {'and '}
             </Text>
           )}
         <Text
-          type={type}
-          backgroundMode={backgroundMode}
-          className={underline ? 'hover-underline' : undefined}
+          type={props.type}
+          backgroundMode={props.backgroundMode}
+          className={props.underline ? 'hover-underline' : undefined}
           onClick={_onUsernameClicked ? () => _onUsernameClicked(u.username) : undefined}
           style={userStyle}
         >
           {u.username}
         </Text>
-        {i !== users.length - 1 &&
-        (!inlineGrammar || users.length > 2) && ( // Injecting the commas here so we never wrap and have newlines starting with a ,
-            <Text type={type} backgroundMode={backgroundMode} style={derivedJoinerStyle}>
+        {i !== props.users.length - 1 &&
+          (!props.inlineGrammar || props.users.length > 2) && (
+            <Text
+              type={
+                props.type // Injecting the commas here so we never wrap and have newlines starting with a ,
```

Comment:
I'm guessing this was prettier but this seems like a weird spot for this comment, could you move it closed to the comma?